packages/react-components/src/templates/__tests__/DashboardPageBody.test.tsx

```diff
@@ -72,6 +74,31 @@ it('hides add links to your work space section when user is not a member of a te
   expect(screen.queryByText(/Add important links/i)).toBeNull();
 });
 
+describe('the past events card', () => {
+  const events = [
+    createEventResponse({}, 0),
```

Comment:
Can't you pass createListEventResponse(3) there ?

Reply:
didn't know it existed, will do that!